### DISCUSSION TITLE
Prism.MAUI Production?

### DISCUSSION THREAD

**Discussion by jjohnnyl** [PrismLibrary/Prism#3049] [.NET MAUI] [answered]
We are wanting to buy a Prism license, but we are wanting to know what your long term support for MAUI will be like? 
The MAUI library still seems to be in pre-release mode still and the documentation regarding MAUI seems to be old.

**Comment by brianlagunas** [comment, marked as answer]
Only Commercial Plus license holders have access to our current builds of Prism.MAUI. Public releases will be much slower compared to valid license holders.  Docs are a work in progress. The docs site may be outdated, but the repo is current. You can always fork the docs repo and run them locally to see the latest docs as they are being worked on.

**Comment by jjohnnyl** [comment]
Thanks Brian, I have signed up to the commercial plus license.

How I can get access to the current MAUI builds? Didn't have anything send through apart from the receipt/invoice.

Thanks

**Reply by brianlagunas** [reply]
Give us some time. We manually process each purchase until we can build our automated system.  I will be emailing you your license key and instructions for getting the latest builds

**Reply by jjohnnyl** [reply]
Awesome, thanks for the quick update!
